src/mariadb/videx/CMakeLists.txt

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) 2006, 2023, Oracle and/or its affiliates.
```

Comment:
let's change the first line to `/* Copyright (c) 2025 Bytedance Ltd. and/or its affiliates` 😂

I think `src/mysql/videx/CMakeLists.txt` has the same problem, we can fix it later.